client/reader/start/main.jsx

```diff
@@ -77,35 +53,32 @@ const Start = React.createClass( {
 	},
 
 	render() {
-		const totalSubscriptions = this.state.totalSubscriptions;
-
-		// Reduce the total subscription count by one for display (exclude the user's own site)
-		const totalSubscriptionsDisplay = totalSubscriptions > 0 ? totalSubscriptions - 1 : 0;
-		const canGraduate = ( this.state.totalSubscriptions > 1 );
+		const followCount = this.props.followCount;
+		const canExit = ( followCount > 0 );
 		const hasRecommendations = this.props.recommendationIds.length > 0;
 		const hasSearchQuery = !! this.props.query;
 
 		return (
 			<Main className="reader-start">
 				{ /* Have not followed a site yet */ }
-				{ ! canGraduate && hasRecommendations &&
+				{ ! canExit && hasRecommendations &&
 					<div className="reader-start__bar is-follow">
 						<span className="reader-start__bar-text">{ this.translate( 'Follow some sites to begin.' ) }</span>
 					</div>
 				}
 
 				{ /* Following at least one or more sites */ }
-				{ canGraduate && hasRecommendations &&
+				{ canExit && hasRecommendations &&
 					<div className="reader-start__bar is-following">
 						<span className="reader-start__bar-text">
 							{
 								this.translate(
-									'You\'re following %(totalSubscriptionsDisplay)d site.',
-									'You\'re following %(totalSubscriptionsDisplay)d sites.',
+									'You\'re following %(followCount)d new site.',
```

Comment:
maybe these should be past tense? "You've followed 1 new site / You've followed n new sites"?